txnkv/transaction/txn.go

```diff
@@ -395,6 +395,22 @@ func (txn *KVTxn) SetTxnSource(txnSource uint64) {
 	txn.txnSource = txnSource
 }
 
+// SetSessionID sets the session ID of the transaction.
+// If the committer is not initialized yet, the function won't take effect.
```

Comment:
For the concern of completeness of the API, do we consider put it to another field temporarily if the committer is not initialized yet, and use it lazily when initializing the committer?

Reply:
It's worth consideration. But I don't want to do it now as it's not needed